src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1824,14 +1824,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         much information as possible will be passed to the compilers.
         -->
 
+    <PropertyGroup>
+      <MSBuildGlobalPropertiesToRemoveFromProjectReferences Condition="'$(PassOutputPathToReferencedProjects)'=='false'">$(MSBuildGlobalPropertiesToRemoveFromProjectReferences);OutputPath</MSBuildGlobalPropertiesToRemoveFromProjectReferences>
+    </PropertyGroup>
+
     <MSBuild
         Projects="@(_MSBuildProjectReferenceExistent)"
         Targets="GetTargetPath"
         BuildInParallel="$(BuildInParallel)"
         Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration); %(_MSBuildProjectReferenceExistent.SetPlatform); %(_MSBuildProjectReferenceExistent.SetTargetFramework)"
         Condition="'%(_MSBuildProjectReferenceExistent.BuildReference)' == 'true' and '@(ProjectReferenceWithConfiguration)' != '' and ('$(BuildingInsideVisualStudio)' == 'true' or '$(BuildProjectReferences)' != 'true') and '$(VisualStudioVersion)' != '10.0' and '@(_MSBuildProjectReferenceExistent)' != ''"
         ContinueOnError="!$(BuildingProject)"
-        RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)">
+        RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)$(MSBuildGlobalPropertiesToRemoveFromProjectReferences)">
```

Comment:
There are several other calls to the MSBuild task in this target and elsewhere where this should be passed.

Reply:
I wasn't sure what qualified as a place to pass it or not pass it, so I put it everywhere we removed GlobalPropertiesToRemove. Does that sound appropriate?